src/RegistryCoordinator.sol

```diff
@@ -480,8 +490,12 @@ contract RegistryCoordinator is
         require(quorumsToAdd.noBitsInCommon(currentBitmap), "RegistryCoordinator._registerOperator: operator already registered for some quorums being registered for");
         uint192 newBitmap = uint192(currentBitmap.plus(quorumsToAdd));
 
-        // Check that the operator can reregister if ejected
-        require(lastEjectionTimestamp[operator] + ejectionCooldown < block.timestamp, "RegistryCoordinator._registerOperator: operator cannot reregister yet");
+        // Check that the operator can reregister 
+        require(
+            lastEjectionTimestamp[operator] + ejectionCooldown < block.timestamp &&
+            lastDeregistrationTimestamp[operator] + deregistrationCooldown < block.timestamp,
```

Comment:
why do we need a deregistrationCooldown if we keep the entire history of keys onchain?

Reply:
Or is 0 a reasonable value for deregistrationCooldown?

Reply:
@gpsanant has suggested this should be a few days. I think something like an hour is more reasonable

Reply:
this...limits the rate at which operators can edit their keys. I don't think it's serving a serious purpose beyond maybe addressing some theoretical worry where an operator keeps changing their key? I guess if you need to search past keys it's potentially problematic if the growth rate of the historical storage isn't limited?